test/unit/infrastructure/adapters/GitCasWarpStateCacheAdapter.test.ts

```
@@ -5,13 +5,71 @@ import ORSet from '../../../../src/domain/crdt/ORSet.ts';
+const mockRootRead = vi.fn(async (): Promise<RootSetState> => ({
+  ref: 'refs/cas/rootsets/git-warp/test-graph/state-cache',
+  headOid: mockRootEntries.length === 0 ? null : 'a'.repeat(40),
+  treeOid: mockRootEntries.length === 0 ? null : 'a'.repeat(40),
+  entries: [...mockRootEntries],
+}));
+const mockRootMutate = vi.fn(async (
+  mutator: (
+    entries: ReadonlyArray<Readonly<RootSetEntry>>,
+  ) => Iterable<RootSetEntry> | Promise<Iterable<RootSetEntry>>,
+): Promise<RootSetMutationResult> => {
+  mockRootEntries = Array.from(await mutator(mockRootEntries));
```

ESLint

[error] Use spread syntax [...iterable] instead of Array.from(iterable) when no mapper function is provided

(e18e/prefer-spread-syntax)